client/audio/audio_qt.cpp

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPLv3-or-later
+// SPDX-FileCopyrightText: Freeciv authors
+// SPDX-FileCopyrightText: Freeciv21 authors
```

Comment:
In my other IWYU work, I've been combining these two into a single line
`// SPDX-FileCopyrightText: Freeciv21 and Freeciv Contributors`